keyboards/matchstickworks/southpad/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u2
+
+# Bootloader selection
+BOOTLOADER = bootloadHID
```

Comment:
This doesn't seem correct for a 32U2.

Reply:
It builds and flashes correctly. I was trying to correct this to atmel-dfu last time on reading a bit more and my test board didn't want to work anymore, so I reverted.

Reply:
If this board uses fresh 32U2s, then it is definitely `atmel-dfu`. You will likely know if it is using BootloadHID - it's a third party bootloader you have to manually ISP flash onto the chip, it does not come like this from the factory. I also don't think it works with USB AVRs since it runs on V-USB, which implements low-speed USB 1.1 support for chips with no hardware USB peripheral.

Reply:
Roger. Let me play around with it a little more and see if I can get it working properly. I might've been trying to make too many changes at once last time I tried.

Reply:
Figured it out, it was my driver load with Zadig. Now we're good.